.circleci/config.yml

```diff
@@ -413,38 +413,38 @@ workflows:
               only:
                 - develop
     jobs:
-      - uptime_monitor_no_auth:
-          name: "uptime_monitor_no_auth_dev"
+      - no_auth_smoke_tests:
+          name: "no_auth_smoke_tests_dev"
```

Comment:
if this is in quotes, does it work without `_`s?

Reply:
It would still work. The `name` key here overrides the default test name in CircleCI, so if we remove the `_`s, we'll have camelCase amongst snake_case, which might look a bit odd.

No strong feelings, I can change them if we want.

Reply:
I was more curious about actual spaces, no biggie